charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -222,6 +237,47 @@ spec:
           mountPath: /etc/kubernetes/etcd-encryption-secret
           readOnly: true
         {{- end }}
+      {{- if .Values.konnectivityTunnel.enabled }}
+      - name: konnectivity-server
+        image: {{ index .Values.images "konnectivity-server" }}
+        command:
+          - /proxy-server
+        args:
+          - --uds-name=/etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
+          - --logtostderr=true
+          - --cluster-cert=/certs/konnectivity-server/konnectivity-server.crt
+          - --cluster-key=/certs/konnectivity-server/konnectivity-server.key
+          - --agent-namespace={{ .Values.konnectivityTunnel.agentNamespace }}
+          - --agent-service-account=konnectivity-agent
+          - --kubeconfig=/etc/srv/kubernetes/konnectivity-server-kubeconfig/kubeconfig
+          - --authentication-audience=system:konnectivity-server
+          - --mode=http-connect
+          - --server-count={{ .Values.konnectivityTunnel.serverCount }}
+          - --server-port={{ .Values.konnectivityTunnel.serverPort }}
+          - --agent-port={{ .Values.konnectivityTunnel.agentPort }}
+          - --admin-port={{ .Values.konnectivityTunnel.adminPort }}
+        initialDelaySeconds: 30
+        timeoutSeconds: 60
+        resources:
+          requests:
+            cpu: 40m
```

Comment:
Should we also specify some memory requests + cpu/memory limits?

Reply:
I  wanted to get an intuition about its usage before doing so because I don't have good values in mind.

Reply:
What about creating a shoot locally and observing it for a few hours and takes those values + some buffer as starting point? We have (H)VPA in place anyways, so it should scale automatically if needed.